src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,79 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+public class Parser {
+    private boolean bye;
```

Comment:
Hi! Also wanted to point out that by the Java standard coding standard, boolean variables/methods should be named to sound like booleans. Perhaps a name like `isBye` instead, or maybe `isTerminal` instead?